pkgs/development/libraries/xvidcore/default.nix

```diff
@@ -35,12 +35,17 @@ stdenv.mkDerivation rec {
     rm $out/lib/*.a
   '';
 
+  # Dependants of xvidcore don't know to look in bin for dependecies. Link them
+  # in lib so other depedants of xvidcore can find the dlls.
```

Comment:
Please give an example

Can the hook be improved https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/setup-hooks/win-dll-link.sh

Reply:
ffmpeg is an example.

I think the hook can be improved. When creating my own windows application, I need to manually copy dlls to bin which is not ideal. See https://github.com/seanybaggins/qt-package-overalys/blob/bcd044bdba9be492e4694d096edb0bb4e9dbb027/nix/overlays/pkgs/qtapp-example/default.nix#L36-L43

Reply:
> I think the hook can be improved. When creating my own windows application, I need to manually copy dlls to bin which is not ideal. See [seanybaggins/qt-package-overalys@`bcd044b`/nix/overlays/pkgs/qtapp-example/default.nix#L36-L43](https://github.com/seanybaggins/qt-package-overalys/blob/bcd044bdba9be492e4694d096edb0bb4e9dbb027/nix/overlays/pkgs/qtapp-example/default.nix#L36-L43)
 
 The `/bin` copies/links should be done by the hook as long as you add the packages to `buildInputs`.